ovn-tester/ovn_tester.py

```diff
@@ -185,6 +185,8 @@ def run_base_cluster_bringup(ovn, bringup_cfg):
     # create ovn topology
     with Context("base_cluster_bringup", len(ovn.worker_nodes)) as ctx:
         ovn.create_cluster_router("lr-cluster")
+        ovn.create_cluster_join_switch("ls-join", ovn.cluster_cfg.gw_net,
```

Comment:
Nit: we don't need to pass `ovn.cluster_cfg.gw_net`, we can take care of the internals in `create_cluster_join_switch`.  But I can fix this up when merging the PR.  Otherwise the rest looks good to me.